README.md

```diff
@@ -1 +1,44 @@
-# mv-awesome-books
\ No newline at end of file
+# HTML-CAPSTONE-PROJECT
+
+> This project is a Microverse exercise on Javascript DOM manipulation, event listener, arrow function, let and const, Javascript Object, Arrays, String interpolation, local storage.
+
+
+![screenshot]()
+
+## Built With
+
+- Semantic HTML
+- JavaScript
+- Github workflow
+- Linters(Lighthouse, stylelint, Webhint, eslint)
+
+## Live Preview
+
+[Github deploy Page]()
+
+## Author
+
+👤 **Abimbola Aminat Buari**
+
+- GitHub: [bimbolabuari](https://github.com/bimbolabuari)
+- Twitter: [bimbolabuari](https://twitter.com/bimbolabuari)
+- LinkedIn: [bimbolabuari](https://linkedin.com/in/bimbolabuari)
```

Comment:
- [ ] **OPTIONAL**
As a team 💪, both of you should leave their accounts on the README file. 🙏